debbuild

```diff
@@ -568,10 +568,12 @@ LINE: while (<$fh>) {
 
     # RPM conditionals - transform to generic form
     if (s/^\s*%if(arch|os)\s+//) {
+      $_ = lc expandmacros($_);
```

Comment:
Please don't assign to `$_`. It's used as input a side effect of a bunch of other operations and means that whoever is reading the code doesn't get a useful variable name. For example, in the next line, the set value is an input to `split`, and not within the map block, but a casual reading wouldn't catch that. Not a blocker because other code nearby already does this and I think what is written will work as intended.

I would also suggest adding parens for all function calls for clarity, again not a blocker.

I would suggest rewriting this as
```
my $expanded = lc(expandmacros($_));
my @args = map { " '%{_$1}' == '$_'" } split(/[\s,]+/, $expanded);
```